wp-plugin-sidekick/wpps-scripts
# Make `phpcs` work with PHP `8`

* Make `phpcs` also work with PHP `8`
* Before, it only worked with `7.3`
* This is now working for PHP `7.3` to `8.2` in https://github.com/studiopress/fse-studio/pull/270:

<img width="530" alt="Screen Shot 2023-01-04 at 5 28 33 PM" src="https://user-images.githubusercontent.com/4063887/210669276-5d3ded53-0c86-4e0b-853f-32e3f4418017.png">

## Testing Instructions
1. `cd wp-content/wpps-scripts`
2. `git fetch && git checkout fix/phpcs-errors`
3. `composer i`
4. `cd ../plugins/fse-studio`
5. `git fetch && git checkout test/phpcs-fix`
6. `npm run lint:php`
7. Expected: it passes
